functions/security/index.js

```diff
@@ -26,7 +26,7 @@ const RECEIVING_FUNCTION = 'myFunction';
 // See https://cloud.google.com/compute/docs/instances/verifying-instance-identity#request_signature
```

Comment:
With this change, can you link the correct metadata server URL documentation?

https://cloud.google.com/functions/docs/securing/function-identity#identity_tokens